core/src/syn/parser/idiom.rs

```diff
@@ -239,6 +239,7 @@ impl Parser<'_> {
 				let graph = ctx.run(|ctx| self.parse_graph(ctx, Dir::In)).await?;
 				Part::Graph(graph)
 			}
+			t!("$param") => Part::Value(Value::Param(self.next_token_value()?)),
```

Comment:
Adding support for a parameter here doesn't just change the `FETCH` clause to support params but changes every place where a idiom is allowed which is a lot of places, most of which I am not sure we want to support having parameters in those positions.

Instead of adding support here for parameters I think a better approach would be to implement it in `core::syn::parser::stmt::parts::try_parse_fetch`. That way the change is limited to just the `FETCH` clause.